src/main/kotlin/lotto/domain/strategy/LottoAutoStrategy.kt

```diff
@@ -0,0 +1,33 @@
+package lotto.domain.strategy
+
+import lotto.domain.Lotto
+import lotto.domain.LottoNumber
+
+class LottoAutoStrategy : LottoGeneratorStrategy {
+
+    private val lottoNumbers = (LOTTO_FIRST_NUMBER..LOTTO_LAST_NUMBER).toList()
+
+    override fun generate(lottoCount: Int): Lotto {
+        return Lotto(
+            (0 until lottoCount)
+                .map { generateLottoNumber() }
+                .toList()
+        )
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요. 😃 
```suggestion
        return Lotto(List(lottoCount) { generateLottoNumber() })
```